frontend/lib/analytics/amplitude.ts

```diff
@@ -14,7 +14,7 @@ import JustfixRoutes from "../justfix-route-info";
 import { NorentRoutes } from "../norent/route-info";
 import { EvictionFreeRoutes } from "../evictionfree/route-info";
 import { USER_ID_PREFIX } from "../../../common-data/amplitude";
-import { LaLetterBuilderRoutes } from "../laletterbuilder/route-info";
+import { LaLetterBuilderRouteInfo } from "../laletterbuilder/route-info";
```

Comment:
It was confusing me to no end that the stuff from route-info.tsx was called Routes and the stuff from routes.tsx was called something else. I changed it to make it make more sense.

Reply:
thank you!!